deployments/_deployments_log_file.json

```diff
@@ -24400,6 +24400,20 @@
           }
         ]
       }
+    },
+    "unichain": {
+      "production": {
+        "1.0.0": [
```

Comment:
is there any reason why the version is `1.0.0` instead of `1.1.0` with new selectors?